charged-ieee/template/main.typ

```diff
@@ -37,9 +38,48 @@ To demonstrate the potential of Typst, we conducted a series of experiments comp
 
 Overall, we believe that Typst represents a significant step forward in the field of scientific writing and typesetting, providing researchers with a valuable tool to streamline their workflow and focus on what really matters: their research. In the following sections, we will introduce Typst in more detail and provide evidence for its superiority over other typesetting systems in a variety of scenarios.
 
-= Methods
-#lorem(90)
+= Methods <sec:methods>
+#lorem(45)
 
-$ a + b = gamma $
+$ a + b = gamma $<formula>
+
+#lorem(80)
+
+#figure(
+  placement: none,
+  image("sun.svg", width: 21%),
```

Comment:
Might be a bit nitpicky from my part, but maybe we can just use a simple circle instead of including an SVG? The official template demo uses the same.